addok/ds.py

```diff
@@ -25,6 +25,9 @@ def remove(self, *keys):
             pipe.delete(key)
         pipe.execute()
 
+    def flush(self):
```

Comment:
`flushdocs` is probably more explicit and consistent with `flushdb` counterpart?